### PR TITLE
Restart from stage

### PR DESCRIPTION
**Pull request by skaspi**
Hi,

This is a part of the following ticket: https://projects.engineering.redhat.com/browse/RHEVM-5503

Please CR + merge

David

### CODE REVIEW

**Comment by KKoukiou** [comment]
@skaspi Don't open to send a new revision. Just force push int he same branch please.
In addition https://github.com/rhevm-qe-automation/rhv-qe-jenkins-library/pull/8/commits/6f8ad11d5cc8b0203d60038aa57ca902a4f9f154 is empty.

This commit https://github.com/rhevm-qe-automation/rhv-qe-jenkins-library/pull/8/commits/7222f6c9bea10c8cae8b08e7262cd13f433669cf should not be there, it's clear that it's wrong

Please cleanup the history and force push here.

**Comment by skaspi** [comment]
@KKoukiou Done

File: vars/other.groovy
Comment (kobihk) [inline vars/other.groovy]: could you please change the "e" to something more readable like stage

Comment (skaspi) [reply vars/other.groovy]: 'e' stands for 'element' . It is kind of convention when iterating through maps/dict keys.
But I will accept your comment

**Review by kobihk — APPROVED** [review]

**Comment by skaspi** [comment]
@KKoukiou Katerina, we are waiting for your approval, so we can proceed with our task.

**Comment by KKoukiou** [comment]
> @KKoukiou Katerina, we are waiting for your approval, so we can proceed with our task.

Please don't bother waiting for extra approvals since you already have one :)
Please merge when this is tested. I am not familiar with how you tested this.

**Pull request merged**
